### DISCUSSION TITLE
Creating URLs that are usable outside of Freeplane that links to a specific document and node

### DISCUSSION THREAD

**Discussion by Koniak** [freeplane/freeplane#1403] [Help] [answered]
Hi,

When combining Freeplane with other tools I find a need to be able to link from other applications to a Freeplane node in a specific Freeplane mindmap.

An example is the tool Obsidian which allows linking to a heading in a specific document as per:
obsidian://open?vault=ObsidianVault1&file=AFile%23AHeading

Ie This URL can be used in any application supporting links and will start the application Obsidian, open the document and go to the given heading. (I use this on Windows 11, but assume this also work on Mac, etc.)

Is there any similar functionality for Freeplance wher I can link from another application directly to a node in a Freeplane document?

**Comment by JJSlote** [comment, marked as answer]
Yes, Freeplane has this capability. Right-click on any node, hover on Copy, and select `Copy Node URI` for the structure. For a large map, I've generated a local HTML file with all of the map's parent nodes. It's then very easy to move new nodes where they belong: select the nodes to be moved, Copy (Ctrl+C), click on a link in the HTML file to direct Freeplane to the destination node, then Alt+V to move the selected nodes.

**Reply by Koniak** [reply]
Thank you for the tip!

That's exactly what I'm looking for.

First tested the generated link in word and it didn't work. But that was because word forces two slash at the beginning (freeplane://....) which you cannot remove. But as you said, testing it in an html page worked like a charm!
